src/test/java/com/amazon/opendistroforelasticsearch/security/InitializationIntegrationTests.java

```diff
@@ -198,6 +198,33 @@ public void testDefaultConfig() throws Exception {
         Assert.assertEquals(HttpStatus.SC_OK, rh.executeGetRequest("", encodeBasicHeader("admin", "admin")).getStatusCode());
     }
 
+    @Test
+    public void testInvalidDefaultConfig() throws Exception {
+        String defaultInitDirectory = System.getProperty("security.default_init.dir");
+        try {
+            System.setProperty("security.default_init.dir", new File("./src/test/resources/invalid_config").getAbsolutePath());
+            final Settings settings = Settings.builder()
+                    .put(ConfigConstants.OPENDISTRO_SECURITY_ALLOW_DEFAULT_INIT_SECURITYINDEX, true)
+                    .build();
+            setup(Settings.EMPTY, null, settings, false);
+            RestHelper rh = nonSslRestHelper();
+            Thread.sleep(10000);
+            Assert.assertEquals(HttpStatus.SC_SERVICE_UNAVAILABLE, rh.executeGetRequest("", encodeBasicHeader("admin", "admin")).getStatusCode());
+
+            System.setProperty("security.default_init.dir", new File(defaultInitDirectory).getAbsolutePath());
+            restart(Settings.EMPTY, null, settings, false);
+            rh = nonSslRestHelper();
+            Thread.sleep(10000);
+            Assert.assertEquals(HttpStatus.SC_OK, rh.executeGetRequest("", encodeBasicHeader("admin", "admin")).getStatusCode());
+        } finally {
+            if (defaultInitDirectory != null) {
+                System.setProperty("security.default_init.dir", new File(defaultInitDirectory).getAbsolutePath());
```

Comment:
It is not necessary to use `new File(defaultInitDirectory).getAbsolutePath()`. `defaultInitDirectory` is prior value of `"security.default_init.dir"` property and it can be used as is.